playbooks/02 - Use Case - Malware Response Using SIEM and EDR Solutions/Investigate Malware Response (Splunk_QRadar_FortiEDR).json

```diff
@@ -73,39 +106,71 @@
+                    "records": ""
+                },
+                "resource": {
+                    "status": "\/api\/3\/picklists\/758925e7-629c-46d8-89db-fb36f5fbe88a"
+                },
+                "_showJson": false,
+                "operation": "Append",
+                "collection": "{{vars.alertIRI}}",
+                "__recommend": [],
+                "collectionType": "\/api\/3\/alerts",
+                "fieldOperation": {
+                    "recordTags": "Append"
+                },
+                "step_variables": []
+            },
+            "status": null,
+            "top": "300",
+            "left": "475",
+            "stepType": "\/api\/3\/workflow_step_types\/b593663d-7d13-40ce-a3a3-96dece928722",
+            "uuid": "12f1214f-5019-4090-b20f-8b7759c66ea4"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Suspicious Endpoint Analysis",
             "description": null,
             "arguments": {
                 "type": "DecisionBased",
                 "input": {
                     "schema": {
-                        "title": "Investigation Summary",
-                        "description": "Please review the following investigation summary of malicious indicators.<\/br>\n\u00a0{% for summary in vars.steps.Hunt_Malicious_Indicator | sort(attribute='indicator_type') %}{{summary.analysis_summary}}\n{% endfor %}\n<br \/>\nPlease provide your approval to block the indicator on the firewall.",
+                        "title": "Manually Analyze Suspicious Endpoint",
+                        "description": "A suspicious malware activity has been detection for the following endpoint, Refer the below details:\n<br \/>\n**Alert ID**  - {{vars.alertID}}\n**Source IP** - {{vars.sourceIP}}\n<br \/>\nThere is no malicious indicators found for this alert. \n<br \/>\nPlease investigate this manually to detemine if this is malicious activity.",
```

Comment:
Detection -->> Detected

Reply:
@cs-sujeet-ahirrao  please incorporate this review comment

Reply:
Done